### ISSUE TITLE
?

### ISSUE THREAD

**Issue by FastZer** [9ggy/Roblox-Gambling-Token-Grabbers#6]
Does this still work? And how does it work? Do i just put it in console?

**Comment by AdamxzV3** [comment]
yes, i have support for all websites in one check mine. this will work too.

**Comment by spearphishing** [comment]
yeah these scripts still work, i'll be updating them soon enough.

**Comment by Flameygod1123** [comment]
how do you run them
